utils/python/CIME/utils.py

```diff
@@ -799,3 +797,20 @@ def transform_vars(text, case=None, subgroup=None, check_members=None, default=N
 def get_my_queued_jobs():
     # TODO
     return []
+
+def wait_for_unlocked(filepath):
```

Comment:
I was using this to try to get around the "busy file" errors I was getting when copying model.exe. I then stumbled upon the "remove it first" technique which seemed to work a lot better. I left this function in in case it would be useful in the future.